graph-viz/app/controllers/IncrementalQuery.java

```diff
@@ -0,0 +1,65 @@
+package controllers;
+
+import utils.PropertiesUtil;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Calendar;
+
+/**
+ * Implement the incremental database query
+ */
+public class IncrementalQuery {
```

Comment:
This class does not extend Controller class so it is not really a controller maybe move it to some other package